package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "hello-rails-front-end",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@reduxjs/toolkit": "^1.8.3",
+    "@testing-library/jest-dom": "^5.16.4",
+    "@testing-library/react": "^13.3.0",
+    "@testing-library/user-event": "^13.5.0",
+    "axios": "^0.27.2",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-redux": "^8.0.2",
+    "react-router-dom": "^6.3.0",
+    "react-scripts": "5.0.1",
+    "web-vitals": "^2.1.4"
+  },
+  "scripts": {
+    "start": "PORT=3001 react-scripts start",
```

Comment:
- [ ] Kindly remove this `Port=3001` from scripts because it actually gives a console error and the app does not start.
- Console error:
![Screenshot 2022-07-13 181819](https://user-images.githubusercontent.com/85113567/178795196-a4607a41-28c7-4d17-b0b2-9461b0cec19a.png)

Reply:
You can just write in your readme file that the user should run the backend on port 3000 and the frontend on a different port 👍

Reply:
Yes, but in the backend I only allowed port 3001 to send requests, that's why, but I already updated the backend to receive from any port, it is less safe but will work on every computer